examples/gradle-pipeline/gradle-pipeline.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+# Define variables.
+GIT_ROOT=$(git rev-parse --show-toplevel)
+DEFAULT_REPOSITORY=$(xxd -l 16 -c 16 -p < /dev/random)
+: "${REGISTRY:=ttl.sh}"
+: "${REPOSITORY:=$REGISTRY/$DEFAULT_REPOSITORY}"
+C_GREEN='\033[32m'
+C_RESET_ALL='\033[0m'
+
+# Install shared tasks.
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/git-clone/0.4/git-clone.yaml
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/gradle/0.3/jib-gradle.yaml
```

Comment:
I think we probably want to vendor the 0.4 version of this task from Tekton Catalog. https://github.com/buildsec/ssf/pull/156 has an example of doing that.

Reply:
Definitely  I pushed up that change.